src/data/faq_de.json

```
@@ -1434,6 +1447,8 @@
                                 "anchor": "tan_procedure",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Aktualisiert am ?????r</b>",
+                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",

```

Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```